### ISSUE TITLE
Update to 1.03 got wrong highestBlock

### ISSUE THREAD

**Issue by usrbad** [masa-finance/masa-node-v1.0#97]
After update to 1.03 and delete "data" folder yesterday, it worked fine and eth.syncing was "false".
Now, when we've got latest block the network says:
```geth attach ipc:$HOME/masa-node-v1.0/data/geth.ipc --exec web3.eth.syncing```
```{
  currentBlock: 4931,
  highestBlock: 3938700,
  knownStates: 0,
  pulledStates: 0,
  startingBlock: 2
}
```
Now highestBlock = 3938700 which seems to be an error.
Lots of ppl have this issue.

**Comment by teslashibe** [comment]
@usrbad we are looking into this we will get back to you asap. Thank you!

**Comment by teslashibe** [comment]
<img width="329" alt="Screen Shot 2022-03-23 at 2 13 08 PM" src="https://user-images.githubusercontent.com/34052452/159796642-0e366b46-cefc-48cb-bd99-d26c228d9c1c.png">
I just tested with a clean install and have synced in around 3 mins with the correct block height. I can only think that somehow the chain data is still local.

**Comment by usrbad** [comment]
> I just tested with a clean install and have synced in around 3 mins with the correct block height. I can only think that somehow the chain data is still local.

Yeah, but as soon as you connect to the wrong peer you'll got this issue.

**Comment by itrocket7** [comment]
I have same problem. Is this problem solved?
